src/error.rs

```diff
@@ -24,13 +24,13 @@ impl From<TonicError> for Error {
 
 impl From<TonicStatus> for Error {
     fn from(error: TonicStatus) -> Self {
-        Error::GrpcError(GrpcError { status: error })
+        Error::GrpcError(GrpcError { _status: error })
     }
 }
 
 #[derive(Debug)]
 pub struct GrpcError {
-    status: TonicStatus,
+    _status: TonicStatus,
 }
```

Comment:
I don't think we should use `_` to prevent warnings for fields like this. If it is unused, but available for use, it should either:
- show up as a clippy warning
- be ignored by clippy